siac/main.go

```
@@ -29,8 +28,10 @@ func apiGet(call string) (*http.Response, error) {
 	if host, port, _ := net.SplitHostPort(addr); host == "" {
 		addr = net.JoinHostPort("localhost", port)
 	}
-
-	resp, err := http.Get("http://" + addr + call)
+	client := &http.Client{}
+	req, _ := http.NewRequest("GET", "http://"+addr+call, nil)
```

Then at least it needs a comment explaining why the error is not checked.